File: IssueTracker_210042122/src/Class/Issue.java
package Class;

import Predefinedvalues.Priority;
import Predefinedvalues.Status;

import java.util.List;

public class Issue {
    private int id;
    private String title;
    private String description;

    private Priority priority;

    private Status status;

    private String creationDate;

    private String lastModifiedDate;

    private String assignedTo;

    private List<String> tags;

    public List<Comment> comments;


    public Issue(int id, String title, String description, Priority priority, Status status, String creationDate, String lastModifiedDate, String assignedTo, List<String> tags, List<Comment> comments) {
        this.id = id;
        this.title = title;
        this.description = description;
        this.priority = priority;
        this.status = status;
        this.creationDate = creationDate;
        this.lastModifiedDate = lastModifiedDate;
        this.assignedTo = assignedTo;
        this.tags = tags;
        this.comments = comments;
    }

    public Issue(String title, String description,Priority priority){
        this.title=title;
        this.description=description;
        this.priority=priority;
        this.status=Status.Open;
    }

    public void addComment(Comment C){
        comments.add(C);

    }



    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return description;
    }

    public Priority getPriority() {
      return  priority;
    }

    public int getId() {
        return id;
    }
}

File: TaskManager/TrelloApp/TrelloApp/Issue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TrelloApp
{
    public class Issue
    {
        public int id;
        public string title;
        public string description;
        public enum priority { Low, Medium, High, Critical };
        public priority priorityType;
        public enum status { Open, InProgress, UnderReview, Resolved };
        public status statusType;

        public int creationDate;
        public int updatedDate;

        public string assignedTo;
        public List<string> tags = new List<string>();
        public List<Comment> comments = new List<Comment>();

        public Issue(int id, string title, string description, priority priority, 
            status status,int creationDate) 
        {
            this.id = id;
            this.title = title;
            this.description = description;
            priorityType = priority;
            statusType = status;
            this.creationDate = creationDate;
        }

        public Issue()
        {
            
        }

        public void addComment(Comment comment)
        {
            comments.Add(comment);
        }
    }
}
